src/components/account.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom'
+
+import { StyledBtns, StyledNav } from '../Styles';
+
+export default function Account() {
+   return (
+      <>
+         <header className="header">
+            <h1>Randomized Character Creator</h1>
+
+            <nav>
+               <StyledNav>
+                  <StyledBtns>
+                     <NavLink to={`/`} id='home link'>
+                        <div className='home btn'>Home</div>
+                     </NavLink>
+                  </StyledBtns>
+                  <StyledBtns>
+                     <NavLink to={`/character-randomizer`} id='randomizer link'>
+                        <div className='randomizer btn'>Randomizer</div>
+                     </NavLink>
+                  </StyledBtns>
+                  <StyledBtns>
+                     <NavLink to={`/about`} id='about link'>
+                        <div className='about btn'>About</div>
+                     </NavLink>
+                  </StyledBtns>
+                  <StyledBtns>
+                     <NavLink to={`/contact`} id='contact link'>
+                        <div className='contact btn'>Contact</div>
+                     </NavLink>
+                  </StyledBtns>
+                  <StyledBtns>
+                     <NavLink to={`/login`} id='login link'>
+                        <div className='login btn'>Login</div>
+                     </NavLink>
+                  </StyledBtns>
+               </StyledNav>
+            </nav>
+         </header>
+
+         <h1>This page is under construction.</h1>
+
+         <footer>
```

Comment:
Same with the footer. If it's going to be reused on all pages you could create a component to be reused